about/index.md

```diff
@@ -41,7 +41,7 @@ MEI and MusicXML share some similarities -- both of them encode music notation (
 
 ## What is the history of MEI?
 
-In 1999, Perry Roland created an XML schema (DTD) for the representation of music notation. Eventually, this DTD became known as MEI, drawing on the same principles that guided the creation of the [Text Encoding Initiative](http://www.tei-c.org/) (TEI). Perry presented his initial work at the first [International Society for Music Information Retrieval](http://ismir.net/conferences.html) in 2000.
+In 1999, Perry Roland created an XML schema (DTD) for the representation of music notation. Eventually, this DTD became known as MEI, drawing on the same principles that guided the creation of the [Text Encoding Initiative](http://www.tei-c.org/) (TEI). Perry presented his initial work at the first [International Society for Music Information Retrieval](https://ismir.net/conferences/) in 2000.
```

Comment:
```suggestion
In 1999, Perry Roland created an XML schema (DTD) for the representation of music notation. Eventually, this DTD became known as MEI, drawing on the same principles that guided the creation of the [Text Encoding Initiative](http://www.tei-c.org/) (TEI). Perry presented [his initial work](http://ismir2000.ismir.net/papers/roland_paper.pdf) at the first [International Society for Music Information Retrieval](https://ismir2000.ismir.net/) in 2000.
```

Reply:
The name of the ISMIR link should be "ISMIR conference".
@musicEnfanthen I wouldn't add a direct link to the paper here, as it is available in the bibliography.

Reply:
Ok, agreed